packages/server/database/migrations/20210527170505-addAdditionalRetroTemplates.ts

```diff
@@ -0,0 +1,199 @@
+import {R} from 'rethinkdb-ts'
+const Redis = require('ioredis')
+
+const createdAt = new Date()
+const redis = new Redis(process.env.REDIS_URL)
+
+const nameToId = (name: string, isTemplate: boolean) => {
+  const cleanedName = name
+    .replace(/[^0-9a-z-A-Z ]/g, '') // remove emojis and apostrophes
+    .split(' ')
+    .map(
+      (name, idx) =>
+        (idx === 0 ? name.charAt(0).toLowerCase() : name.charAt(0).toUpperCase()) + name.slice(1)
+    )
+    .join('')
+    .trim()
+  return `${cleanedName}${isTemplate ? 'Template' : 'Prompt'}`
+}
+
+const makeTemplate = (name: string) => ({
+  createdAt,
+  id: nameToId(name, true),
+  isActive: true,
+  name,
+  orgId: 'aGhostOrg',
+  scope: 'PUBLIC',
+  teamId: 'aGhostTeam',
+  type: 'retrospective',
+  updatedAt: createdAt,
+  isStarter: true
+})
+
+type PromptInfo = {
+  question: string
+  description: string
+  groupColor: string
+  templateId: string
+  sortOrder: number
+}
+
+const makePrompt = (promptInfo: PromptInfo) => {
+  const {question, description, groupColor, templateId, sortOrder} = promptInfo
+  return {
+    createdAt,
+    description,
+    groupColor,
+    id: nameToId(question, false),
+    isActive: true,
+    question,
+    sortOrder,
+    teamId: 'aGhostTeam',
+    templateId,
+    title: question,
```

Comment:
It doesn't look like we're making use of `title` in prompts - I can't see it being requested anywhere. I've followed [this format](https://github.com/ParabolInc/parabol/blob/a6f7d3153947b4f5cd70ef48912bbdf00b60bb94/packages/server/database/migrations/20180904093953-addRetroTemplates.js#L54) of using the question as the title based on the [type description](https://github.com/ParabolInc/parabol/blob/03facdd6bf959cc0e8319adc61d078ccbcdb1f6b/packages/server/graphql/types/ReflectPrompt.ts#L50)

Reply:
true, i think it's an artifact that we never refactored away, we should probably do that...

Reply:
Cool, I've created an [issue](https://github.com/ParabolInc/parabol/issues/5009)